fastlane/Fastfile

```diff
@@ -0,0 +1,114 @@
+fastlane_version "2.63.0"
+
+skip_docs
+
+desc "Run the tests"
+lane :test do
+  scan(scheme: "JOSESwift")
+end
+
+desc "Lint Swift files"
+lane :lint do
+  # Todo: Don't ignore exit status (#98)
+  swiftlint(ignore_exit_status: true)
+end
+
+desc "Format Swift files"
+lane :format_code do
+  swiftlint(mode: :autocorrect)
+end
+
+desc "Bumps the version number"
+lane :bump do |options|
+  bump_type = options[:type]
+  version = options[:version]
+
+  new_version_xcode
+  new_version_podspec
+
+  case bump_type
+  when "major", "minor", "patch"
+    new_version_xcode = increment_version_number(bump_type: bump_type)
+    new_version_podspec = version_bump_podspec(path: "JOSESwift.podspec", bump_type: bump_type)
+  else
+    if version.nil?
+      new_version_xcode = increment_version_number
+      new_version_podspec = version_bump_podspec(path: "JOSESwift.podspec")
+    else
+      new_version_xcode = increment_version_number(version_number: version)
+      new_version_podspec = version_bump_podspec(path: "JOSESwift.podspec", version_number: version)
+    end
+  end
+
+  if new_version_xcode == new_version_podspec
+    UI.success "Bumped version number to #{new_version_xcode}"
+  else
+    UI.user_error!("Xcode and .podspec version mismatch: #{new_version_xcode} != #{new_version_podspec}")
+  end
+
+  new_version_xcode
+end
+
+desc "Prepares a release by bumping version, formatting code, running tests, setting up a release preparation branch"
+lane :prepare do |options|
+  bump_type = options[:type]
+  version
+  branch_name
```

Comment:
I think you can remove all three variables above completely now 👍

Reply:
Right! Feels like JS now. 😅